pkg/archive/changes.go

```diff
@@ -386,7 +386,21 @@ func ExportChanges(dir string, changes []Change) (Archive, error) {
 		// during e.g. a diff operation the container can continue
 		// mutating the filesystem and we can see transient errors
 		// from this
+	changesLoop:
 		for _, change := range changes {
+			for _, exclude := range excludes {
+				exclude := filepath.Join("/", exclude)
```

Comment:
Need some help following the flow here.  First, to level-set, 'changes' is not relative to 'dir' but 'excludes' is, right?

If so, I'm not sure I understand the first if-stmt below. When would 'exclude' ever match change.Path since 'exclude' should be a relative path from the root of the FS, while change.Path should include the FS path on the host, right?